### PR TITLE
remove expedited SSL logo as no longer used

### PR DESCRIPTION
**Pull request by rachaelkiteme**
This pr removes expedited SSL Logo as no longer used we are now using `letsencrypt`, fixes #153282531

### CODE REVIEW

**Comment by tansaku** [comment]
Thanks @rachaelkiteme - I think we also need to remove the associated test:

```
 features/admin/acknowledging_contributors_site.feature:15
```

**Comment by rachaelkiteme** [comment]
Its okay let me do it

Regards
Rachael

File: app/views/layouts/_footer2.html.erb [hunk 1]
```diff
@@ -11,10 +11,7 @@
         </a>
         &nbsp; &nbsp;
       <% end %>
-      <a href="https://www.expeditedssl.com/simple-ssl-scanner/scan?target_domain=www.harrowcn.org.uk" target="_blank">
-        <img width='70px' height='40px' alt="Ssl-secure-badge" src="https://www.expeditedssl.com/ssl-secure-badge.png" />
-      </a>
-    </small>
+          </small>
```
Comment (sigu) [inline app/views/layouts/_footer2.html.erb]: Check on the indentation, its a bit too much on the inside :smile:

File: features/admin/acknowledging_contributors_site.feature [hunk 2]
```diff
@@ -9,7 +9,7 @@ Feature: Acknowledging Contributors
     Given I visit the <page> page
     Then I should see the "Agile Ventures" image linked to contributors
 #    And I should see the "Do-it" image linked to "https://do-it.org/"
-    And I should see the "Ssl-secure-badge" image linked to "https://www.expeditedssl.com/simple-ssl-scanner/scan?target_domain=www.harrowcn.org.uk"
+    scan?target_domain=www.harrowcn.org.uk"
```
Comment (sigu) [inline features/admin/acknowledging_contributors_site.feature]: Seems you forgot to remove this line `scan?target_domain=www.harrowcn.org.uk"`

Comment (rachaelkiteme) [reply features/admin/acknowledging_contributors_site.feature]: Yeah true. Noted

**Pull request merged**